<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>592</width>
        <height>331</height>
       </rect>
      </property>
      <attribute name="label">
       <string>初始化</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>461</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="borrowingInitial">
          <property name="text">
           <string>借款记录初始化</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="incomeInitial">
          <property name="text">
           <string>收入记录初始化</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="spendingInitial">
          <property name="text">
           <string>支出记录初始化</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="userinfoInitial">
          <property name="text">
           <string>账户余额初始化</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>各个账户余额清零</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>所有收入记录清零</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>所有支出记录清零</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>所有借款记录清零</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>592</width>
        <height>331</height>
       </rect>
      </property>
      <attribute name="label">
       <string>账户设置</string>
      </attribute>
      <widget class="QToolBox" name="toolBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>541</width>
         <height>291</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>28</height>
         </rect>
        </property>
        <attribute name="label">
         <string>设置余额</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>441</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="3">
           <widget class="QLabel" name="label_money">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>账户：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_account">
            <item>
             <property name="text">
              <string>微信</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>银行卡</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>一卡通</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>现金</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>其他</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>负债</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>当前余额：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>修改余额：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_money">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton_banlance">
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>541</width>
          <height>249</height>
         </rect>
        </property>
        <attribute name="label">
         <string>设置密码</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>0</y>
           <width>481</width>
           <height>271</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_old"/>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>显示密码</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>请输入原密码：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>请输入新密码：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_password">
            <property name="text">
             <string>确认修改</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>密码长度不限</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="lineEdit_new"/>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>显示密码</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>帮助</string>
      </attribute>
      <widget class="QToolBox" name="toolBox_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>511</width>
         <height>321</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>28</height>
         </rect>
        </property>
        <attribute name="label">
         <string>关于记录：</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>471</width>
           <height>191</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;在填写完日期，金额，操作账户，分类与标签等信息后，点击保存，即可保存该条记录啦~&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>28</height>
         </rect>
        </property>
        <attribute name="label">
         <string>关于查询：</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>0</y>
           <width>451</width>
           <height>191</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;在限定查询的日期范围后，即可开始检索该日期范围内的所有记录。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;此外，还可搜索指定的账户与指定的分类~&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;双击某条记录即可进入详情页面。详情页面中可对该条记录进行修改或删除~&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>28</height>
         </rect>
        </property>
        <attribute name="label">
         <string>关于统计：</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>461</width>
           <height>201</height>
          </rect>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;点击“账户现有资产分布图”按钮后，即可生成各个账户资产的饼状统计图。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;选择完统计的日期范围后，即可生成五种饼状统计图表：&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    分类支出：按照支出的标签来进行统计。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    账户支出：按照进行支出操作的账户来进行统计。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    分类收入：按照收入的标签来进行统计。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    账户收入：按照进行收入操作的账户来进行统计。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    负债分布：按照借出者的分类来进行统计。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>返回</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spendingInitial</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>on_spendingInitial_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>incomeInitial</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>on_incomeInitial_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>borrowingInitial</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>on_borrowingInitial_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userinfoInitial</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>on_userinfoInitial_cliked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_account</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Settings</receiver>
   <slot>on_comboBox_account_currentIdexChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_incomeInitial_cliked()</slot>
  <slot>on_spendingInitial_cliked()</slot>
  <slot>on_borrowingInitial_cliked()</slot>
  <slot>on_modifyInitial_cliked()</slot>
  <slot>on_userinfoInitial_cliked()</slot>
  <slot>on_comboBox_account_currentIdexChanged()</slot>
 </slots>
</ui>
